bugwarrior/config.py

```diff
@@ -185,7 +185,12 @@ def get_taskrc_path(conf, main_section):
 
 
 def get_data_path():
-    return os.path.expanduser(os.getenv('TASKDATA', '~/.task'))
+    path = os.getenv('TASKDATA', '~/.task')
+    if conf.has_option(main_section, 'taskdata'):
+        path = conf.get(main_section, 'taskdata')
```

Comment:
That's a function parameter which doesn't exist in this function. Travis tests failed due to `NameError: global name 'conf' is not defined`.